src/components/UserNotes.tsx

```diff
@@ -1,447 +1,450 @@
-// UserNotes
 import { MDXEditor } from '@mdxeditor/editor';
 import '@mdxeditor/editor/style.css';
 import React, { useEffect, useState } from 'react';
 import { userNotesAPI } from '../services/api';
 
 // Import all the plugins we need
 import {
-  BoldItalicUnderlineToggles,
-  CreateLink,
-  InsertImage,
-  InsertTable,
-  ListsToggle,
-  UndoRedo,
-  codeBlockPlugin,
-  codeMirrorPlugin,
-  diffSourcePlugin,
-  frontmatterPlugin,
-  headingsPlugin,
-  imagePlugin,
-  linkDialogPlugin,
-  linkPlugin,
-  listsPlugin,
-  markdownShortcutPlugin,
-  quotePlugin,
-  sandpackPlugin,
-  tablePlugin,
-  thematicBreakPlugin,
-  toolbarPlugin,
+BoldItalicUnderlineToggles,
```

Comment:
I don't understand why the whitespace before these lines disappears.

If that's a linter rule, it's a strange one.

Reintroduce the whitespace (not even sure if it's tabs or spaces, but whichever it was originally), and this PR will be a lot more manageable.